src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -3435,7 +3435,7 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "RecycleBin_deleteAllButtonTitle" = "Alle endgültig löschen";
 
-"RecycleBin_expirationDateTime" = "wird am %@ um %@ endgültig gelöscht";
+"RecycleBin_expirationDateTime" = "wird am %@ um %@ Uhr endgültig gelöscht";
```

Comment:
We don't add Uhr in our dates on iOS. We need to be consistent with the format. If we decide we are doing it then we need to do it everywhere.